src/docs/spring/spring-web.adoc

```diff
@@ -39,19 +39,19 @@ management.metrics.distribution:
     percentiles-histogram[http.server.requests]: true <1>
     sla[http.server.requests]: 10ms, 100ms
 ----
-<1> If percentile approximations based on histograms are supported by your monitoring system, prefer this _instead_ of the `percentiles` option.
+<1> If percentile approximations based on histograms are supported by your monitoring system, prefer this approach _instead_ of the `percentiles` option.
 
-=== Only timing endpoints you mark with @Timed
+=== Only Timing Endpoints You Mark with @Timed
```

Comment:
I needed to re-read this a few times to understand. Probably because English is my third language but to me "Timing Only The Endpoints That You Mark with @Timed" is easier to understand.

Reply:
Not just you. Good catch. Thanks.